File: JJB/231129/외계행성의_나이.py
def solution(age):
    answer = ''
    
    # age = [int(x) for x in str(age)]
    # age = map(lambda c : chr(c+ord('a')), age)
    # answer = ''.join(age)
    
    while age != 0:
        answer = chr(age%10 + ord('a')) + answer
        age //= 10
    
    return answer

File: JJB/231129/외계행성의_나이.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

string solution(int age) 
{
    string answer = "";
    
    while(age != 0)
    {
        // answer.push_back(char(age%10+97));
        char temp = age % 10 + 'a';
        answer = temp + answer;
        age /= 10;
    }
    // reverse(answer.begin(), answer.end());
    
    return answer;
}